src/Microsoft.AspNetCore.OData/Extensions/Endpoint/ODataEndpointSelector.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+#if !NETSTANDARD2_0
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNet.OData.Interfaces;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc.Abstractions;
+using Microsoft.AspNetCore.Routing.Matching;
+
+namespace Microsoft.AspNet.OData.Extensions
+{
+    /// <summary>
+    /// A service that is responsible for the final OData Endpoint selection decision.
+    /// This selector is used to resolve the duplicated Endpoint selection.
+    /// </summary>
+    internal class ODataEndpointSelector : EndpointSelector
+    {
+        private EndpointSelector _innerSelector;
```

Comment:
>_innerSelector [](start = 33, length = 14)

I am unclear on what does the coding style guidelines say about using _ for private members. It was the old naming convention and I see that not all classes use this. We should discuss what is the recommendation, since this convention is different in ODL as well.